_layouts/default.html

```diff
@@ -42,44 +42,43 @@
     <script>console.log('Welcome to Project Jupyter! Explore the various tools available and their corresponding documentation. If you are interested in contributing to the platform, please visit the community resources section at https://jupyter.org/community.html.')</script>
   </head>
   <body>
-    <nav class="navbar navbar-fixed-top" role="navigation" id="navbar" >
-        <div class="navbar-inner">
-        <div class="container">
-            <!-- Brand and toggle get grouped for better mobile display -->
-            <div class="navbar-header">
-                <button type="button" class="navbar-toggle" data-bs-toggle="collapse" data-bs-target="#bs-example-navbar-collapse-1">
-                    <span class="sr-only">Toggle navigation</span>
-                    {% if page.url == '/' or page.url == '/index.html' %}
-                      <span class="icon-bar white-icon-bar"></span>
-                      <span class="icon-bar white-icon-bar"></span>
-                      <span class="icon-bar white-icon-bar"></span>
-                    {% else %}
-                      <span class="icon-bar"></span>
-                      <span class="icon-bar"></span>
-                      <span class="icon-bar"></span>
-                    {% endif %}
-                </button>
-                <a class="navbar-brand" href="{{ site.baseurl }}/index.html">
-                    <img id="jupyter-nav-logo" class="navbar-logo" src="{{ site.baseurl }}/assets/nav_logo.svg" alt="Jupyter logo">
-                </a>
-            </div>
-            <div class="blur"></div>
-            <!-- Collect the nav links, forms, and other content for toggling -->
-            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-                <ul class="nav navbar-nav navbar-right">
-                    {%- for nav in layout.nav -%}
-                    <li>
-                        <a href="{{nav.url}}"
-                           {%- if nav.newpage %}target="_blank" rel="noopener noreferrer"{% endif %}
-                           class="tab {% if nav.url == page.url %}navbar-active{% endif %}">
-                           {{nav.title}}
-                       </a>
-                    </li>
-                    {%- endfor %}
-                </ul>
-            </div>
+    <nav class="navbar" role="navigation">
+        <div class="navbar-header">
+            <a class="navbar-brand" href="/">
+               <picture>
```

Comment:
This is cool, I had no idea this existed in HTML

Reply:
There's a very nice article on it here https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images